src/simple_treeview.rs

```diff
@@ -0,0 +1,101 @@
+//! # TreeView Sample
+//!
+//! This sample demonstrates how to create a TreeView with either a ListStore or TreeStore.
+
+extern crate glib;
+extern crate gtk;
+
+use gtk::prelude::*;
+use gtk::{
+    CellRendererText, Label, ListStore, TreeView, TreeViewColumn,
+    Window, WindowPosition, WindowType
+};
+
+fn append_column(tree: &TreeView, id: i32) {
+    let column = TreeViewColumn::new();
+    let cell = CellRendererText::new();
+
+    column.pack_start(&cell, true);
+    // We associate the view's column with the model's `id` column.
+    column.add_attribute(&cell, "text", id);
+    tree.append_column(&column);
+}
+
+fn create_and_fill_model() -> ListStore {
+    // We create a model with two rows.
+    let store = ListStore::new(&[String::static_type(), u32::static_type()]);
```

Comment:
Still puzzled by the order. I've never seen a list with the "№" column on the right-hand side.